app/locales/en.json

```diff
@@ -109,5 +112,8 @@
     "what_does_this_mean": "What does this mean?",
     "what_if_no_symptoms_para": "If you have no symptoms but still would like to be tested you can go to your nearest testing site.\n\nIndividuals who don't exhibit symptoms can sometimes still carry the infection and infect others. Being careful about social distancing and coming in contact with large groups or at risk individuals (the elderly, those with significant other medical issues) is important to manage both your risk and the risk to others.",
     "what_if_no_symptoms": "What if I’m not showing symptoms?"
+  },
+  "markdown": {
```

Comment:
This is going to probably cause inconsistency issues with translations (people not properly escaping etc)..

Move it to an actual file, then we can upload it to Lokalise for translation, and in the app we can select the appropriate translation.